packages/react-components/react-alert/src/stories/AlertAction.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+
+import { DismissCircleRegular } from '@fluentui/react-icons';
+
+import { Alert } from '../index';
+
+export const Action = () => (
+  <div style={{ display: 'flex', flexDirection: 'column', gap: '10px' }}>
+    <Alert
+      intent="success"
+      action={{
+        appearance: 'transparent',
+        children: 'Undo',
+      }}
+    >
+      Message sent
+    </Alert>
+    <Alert
+      intent="error"
+      action={{
+        appearance: 'transparent',
+        icon: <DismissCircleRegular fontSize="16px" />,
```

Comment:
```suggestion
        icon: <DismissCircleRegular />,
```

If we will use scalable icons we can omit `fontSize`